patterns/2-structured/release-process.md

```diff
@@ -0,0 +1,64 @@
+## Title
+
+Standard Release Process
+
+## Patlet
+
+Teams are reluctant to use InnerSource projects that they are unfamiliar with when they are unclear about the release process.
```

Comment:
I think it might not so much be exactly the maturity and longevity but rather how well the project is supported; i.e., if a problem is found and if they contribute back a fix, how (and how quickly) would that fix appear in a supported release. So, maybe: Teams are reluctant to adopt an InnerSource project when they are unclear about how quickly fixes might appear in a supported release. (Maturity and longevity are other qualities of IS projects that could impact adoption, but this pattern seems to speak more towards active support rather than code maturity/quality). Just a thought.

Reply:
Interesting thought. 

Both seem related though i.e. 
* a potential consumer sees that the project is actively supported (through regular releases with good release notes)
* => therefore they assume that the project has reached a certain maturity and that they can rely on it for an extended period (longevity)

@dgrizzanti what do you think? What is the thought process that a team goes through before they adopt the project?

Reply:
I think maturity and longevity are factors when looking to adopt a project, but as @NewMexicoKid mentioned, this pattern wasn't intended to speak to that. Maturity more likely, but not longevity.

I'm going to update the language to include a note about maturity and also try to work in what was mentioned about being "unclear about how quickly fixes might appear in a supported release."